src/Microsoft.ML.StandardLearners/Standard/MultiClass/MetaMulticlassTrainer.cs

```diff
@@ -116,5 +161,41 @@ public override TPred Train(TrainContext context)
                 return pred;
             }
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="inputSchema"></param>
+        /// <returns></returns>
+        public SchemaShape GetOutputSchema(SchemaShape inputSchema)
+        {
+            Host.CheckValue(inputSchema, nameof(inputSchema));
+
+            // Special treatment for label column: we allow different types of labels, so the trainers
+            // may define their own requirements on the label column.
+            if (LabelColumn != null)
+            {
+                if (!inputSchema.TryFindColumn(LabelColumn.Name, out var labelCol))
+                    throw Host.Except($"Label column '{LabelColumn.Name}' is not found");
```

Comment:
>Except [](start = 31, length = 6)

`ExceptSchemaMismatch`

Reply:
I thought ExceptSchemaMismatch is when there is a type mismatch. This is checking for existence?

---
In reply to: [217234801](https://github.com/dotnet/machinelearning/pull/865#discussion_r217234801) [](ancestors = 217234801)